source/issues/overview.rst

```diff
@@ -1,126 +1,158 @@
 .. _overview.rst:
 
+OnDemand's Known Issues
+=======================
+
 .. note::
-	We take community requests and troubleshoot as well on our discourse at https://discourse.osc.edu/
+    We take community requests and troubleshoot as well on our Discourse at https://discourse.osc.edu/.
 
-File Explorer
-=============
+.. File explorer
 
-Common Issues
--------------
+- `File names with XML or HTML break display`_
+- `Uploading large files is not reliable`_
+- `Downloading forbidden directories fails silently`_
+- `Viewing large files causes problems`_
+- `File upload may leave an incomplete file`_
+- `Incorrect link in tree root`_
 
-		When trying to use certain HTML characters your file name will appear incorrectly. 
-		An issue like this could be resolved by not using the HTML characters. Below are screenshots of what may happen if you do.
+.. Dashboard
 
-		.. figure:: /images/file-names-bad-1.png
-		   :align: center
+- `Rebuild app action in developer view may fail`_
```

Comment:
Lets just remove the "rebuild app action in developer view". This doesn't affect enough (maybe any) people outside of OSC.